.toml

```diff
@@ -0,0 +1,16 @@
+[tool.black]
```

Comment:
What is this file for? Could be useful to have a comment for curious future viewers.

Reply:
Thanks, I added a comment. This file is basically to specify which files to format in black and also includes parameters for black